web/client/localConfig.json

```diff
@@ -419,7 +419,7 @@
                   "EPSG:3857"
                 ],
                 "allowedRoles": [
-                  "ADMIN"
+                  "ALL"
```

Comment:
why this change? please, revert this

Reply:
Sorry was testing using this will revert this back